assets/gabe-idle-run-animation-events.trickfilm.ron

```diff
@@ -0,0 +1,28 @@
+{
+    "idle": (
+        keyframes: KeyframesVec([0]),
+        /* optional */ keyframe_timestamps: [0.0], /* Must be same size as keyframes, if provided */
+        duration: 0.1, /* Must be greater than max(keyframe_timestamps), if provided */
+    ),
+    "run": (
+        keyframes: KeyframesRange((start: 1, end: 7)),
+        duration: 0.6, /* Must be greater than max(keyframe_timestamps), if provided */
+        events: {
+            0: {
+                "events::SampleEvent": (
+                    msg: "start",
+                )
+            },
+            3: {
+                "events::SampleEvent": (
+                    msg: "middle",
+                )
+            },
+            5: {
+                "events::SampleEvent": (
+                    msg: "end",
+                )
+            },
+        }
```

Comment:
Man this is really nice, it's pretty much exactly what I had in mind as well. I unfortunately won't be able to make much use of in the near future (so I can't test the API) because I am auto-generating my trickfilm files right now and it would take a lot of effort to get the events in there as well. Though I am looking forward to be using (and testing) this!

Reply:
I was hoping you would use and test it for me :D. Guess I need to start up some project making use of this myself.